app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:CommentsCounter)
+  end
```

Comment:
- [ ] **Optional**: Add "Private" on top of this block. This is because you used `after_save` to call the `#update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method `#private`

- Kindly open this 👉 [Link to learn more](https://api.rubyonrails.org/v7.0.4/classes/ActiveRecord/Callbacks.html)

Reply:
hey thank you for your advise, I will apply it